ec/src/group/wnaf.rs

```diff
@@ -0,0 +1,39 @@
+extern crate alloc;
+use crate::Group;
+use alloc::vec::Vec;
+
+pub fn wnaf_mul<G: Group>(table: &[G], scalar_wnaf: &[i64]) -> G {
```

Comment:
Can this function have some comments specifying the algorithm being used for future readers. e.g. just writing out the expected form of the scalar wnaf, and the equation its computing.